terminus-ui/cohort-date-range/testing/src/test-components.ts

```diff
@@ -0,0 +1,88 @@
+// tslint:disable: component-class-suffix
+import {
+  Component,
+  NgModule,
+  ViewChild,
+} from '@angular/core';
+import {
+  FormBuilder,
+  FormsModule,
+  ReactiveFormsModule,
+} from '@angular/forms';
+import {
+  TsDateRangeComponent,
+  TsDateRangeModule,
+} from '@terminus/ui/date-range';
+
+import { CommonModule } from '@angular/common';
+import { NoopAnimationsModule } from '@angular/platform-browser/animations';
+import { TsCohortDateRangeModule } from '@terminus/ui/cohort-date-range';
+import { createDateRangeGroup } from '@terminus/ui/date-range/testing';
+
+
+@Component({
+  template: `
+      <ts-cohort-date-range
+        [allowCustomDates]="allowCustomDates"
+        [cohorts]="dateCohorts"
+      ></ts-cohort-date-range>
+  `,
+})
+
+// tslint:disable-next-line:component-class-suffix
```

Comment:
This is in two spots (here and top) but isn't needed. You're only seeing that issue in your editor as it doesn't understand our different files. But running our linter will show it's not a real issue. Fingers crossed eslint/tslint gets stable soon